tasks/fork/maintain-rate.ts

```diff
@@ -44,8 +44,7 @@ HyperdriveDeployBaseTask(
                 }))!,
             );
             let stethBalanceIncrease =
-                (stethCurrentBalance * BigInt(interval)) /
-                (365n * 24n * rateFraction);
+                stethCurrentBalance / (365n * 24n * rateFraction);
```

Comment:
Two things:

(1) The `rateFraction` logic seems somewhat convoluted. What's the benefit of dividing by `1 / rate` instead of just multiplying by  the rate?
(2) To make this configurable (which is the point of having `interval` be a variable), why don't we calculate this as:

```
stethBalanceIncrease  = (stethCurrentBalance * BigInt(interval) * rate) / (365n * 24n)
```

Reply:
Still reads much easier, but had add a bit of logic to account for bigint math